content/en/apps/reference/tasks.md

```diff
@@ -21,7 +21,7 @@ Tasks are configured in the `tasks.js` file. This file is a JavaScript module wh
 
 | property | type | description | required |
 |---|---|---|---|
-| `name`| `string` | A unique identifier for the task. Used for querying task completeness. | yes, unique |
+| `name`| `string` | A unique identifier for the task. Used for querying task completeness. | no |
```

Comment:
Our [task schema validation](https://github.com/medic/medic-conf/blob/d52eb7be4c5c71e600235e41705bff8f6bfa8c3a/src/lib/validate-declarative-schema.js#L97) lists name as a required field so medic-conf will fail if this isn't specified.

Reply:
Ah, seems as though that changed with medic-conf/pull/275, likely so that tasks can be tracked from a data analysis perspective:

> This change adds two additional constraints to the declarative configuration schema:
> 
> - Task.name is now a required unique attribute
> - Task.event[n].id is now a required unique attribute when there is more than one event

I will clarify in this PR to match what is existing. We can revisit the value of these fields for the next set of improvements to the declarative config.